pilot/pkg/serviceregistry/mcp/controller.go

```diff
@@ -129,6 +129,7 @@ func (c *controller) Apply(change *sink.Change) error {
 	}
 
 	kind := s.Resource().GroupVersionKind()
+	names := map[string]struct{}{} // If non-incremental, we use empty names to indicate all.
```

Comment:
how about s/names/resourcesUpdated

Reply:
Good. As later we may refactor the string key to a struct, `names` does not cover that meaning.